src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,6 +23,7 @@ public class ModelManager implements Model {
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+    private final ObservableList<Person> targetPersonList;
```

Comment:
(Correct me if I'm wrong) From what I understand, `targetPersonList` should only ever contain zero or one persons. 

It would be better if we subclassed `ObservableList` to enforce this.

Reply:
Yes I agree with your suggestion and will subclass it.